script.bluetooth.delay/resources/settings.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+
+<settings>

```

Comment:
can you use the new style settings? https://kodi.wiki/view/Add-on_settings_conversion

You can do this automatically see first post of https://forum.kodi.tv/showthread.php?tid=350135

Reply:
With the new style settings the addon gets stuck (in the debug log nothing appears).
it's necessary?